x-pack/platform/plugins/shared/ml/public/application/contexts/ml/use_job_selection_flyout.tsx

```diff
@@ -63,6 +67,15 @@ export function useJobSelectionFlyout() {
         try {
           flyoutRef.current = overlays.openFlyout(
             <KibanaContextProvider services={services}>
+              <EuiFlyoutHeader hasBorder>
```

Comment:
rather than moving this header out of `JobSelectorFlyoutContent` where it lives alongside the body and footer, could we passed `flyoutTitleId` in as a prop?

Reply:
@jgowdyelastic  That's a very good question. On the other hand, it's quite strange if we pass `flyoutTitleId`, but at the same time the users of the component won't have a way to override the value for `flyoutTitle`.
I think current option looks more maintainable, but if you insist, I'll change it.

Reply:
> but at the same time the users of the component won't have a way to override the value for flyoutTitle.

IMO the consumer of this flyout should not be able to change the title. 
The title, body and footer are all related and make up the content of the flyout.

With #230595, I was actually in the process of writing a comment there. I'm not sure about passing in title components rather than just passing in a (possibly optional) ID to append to the title. I think the title should be part of the component. By making it a prop, you're encouraging consumers to customise it, which is not what we want for these particular components.

Reply:
ok